specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/applicationGateway.json

```diff
@@ -3179,6 +3179,10 @@
           "type": "boolean",
           "description": "Whether FIPS is enabled on the application gateway resource."
         },
+        "networkIsolationEnabled": {
```

Comment:
>networkIsolationEnabled [](http://example.com/codeflow?start=9&length=23)

Prefer enum over boolean, e.g. networkIsolation: { None, Active, ... } or your better variation.

Reply:
The networkIsolationEnabled property takes only true/false values.